docs/user/hbase/kerberos.rst

```diff
@@ -0,0 +1,87 @@
+                             Kerberos
```

Comment:
Could you please merge in my fixes to the docs formatting https://github.com/geoHeil/geomesa/pull/2 
Thanks